modules/RTC/JitsiTrack.js

```diff
@@ -174,6 +175,13 @@ export default class JitsiTrack extends EventEmitter {
         }
     }
 
+    /**
+     * Returns the video type (camera or desktop) of this track.
+     */
+    getVideoType() {
```

Comment:
If this type will be used only for the messages to the bridge I'd add "bridge" to the name. Also if you don't plan to be public I'd add "_".

Is this the case?

Reply:
> If this type will be used only for the messages to the bridge I'd add "bridge" to the name. Also if you don't plan to be public I'd add "_".
> 
> Is this the case?

It doesn't have to be specific to the bridge message, I made it generic enough so that it can be used for getting the videoType. It is a public method and I am calling that instead of using `track.videoType`.

Reply:
What I'm worried about is that currently we use `track.videoType` everywhere including jitsi-meet and also `track.videoType` can't be `'none'` for local video track. IMHO it gets a little confusing whether to use `track.videoType` or `track.getVideoType()` and what result to expect.

Reply:
I can modify it to return the `videoType` and add the muted check wherever the bridge message is sent. What should it return for the audio tracks though ? Is `none` acceptable there ?